ui/packages/atlasmap/src/Atlasmap/Atlasmap.tsx

```diff
@@ -86,8 +82,10 @@ export const Atlasmap: FunctionComponent<IAtlasmapProps> = ({
     onCreateMapping,
   } = useAtlasmap();
 
+  const isSourceRef = useRef<boolean>(true);
```

Comment:
Please no refs unless strictly necessary. Have distinct dialogs for sources and targets instead.

Reply:
Refactored to remove useRef reference.  Will engage @gashcrumb to review to not bug you on PTO!